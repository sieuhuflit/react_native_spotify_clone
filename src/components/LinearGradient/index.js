import LinearGradient from './LinearGradient';

export default LinearGradient;
