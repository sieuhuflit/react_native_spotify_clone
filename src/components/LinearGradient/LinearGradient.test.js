import React from 'react';
import { shallow } from 'enzyme';
import LinearGradient from './LinearGradient';

const setup = propOverrides => {
  const props = Object.assign({}, propOverrides);
  const wrapper = shallow(<LinearGradient {...props} />);
  return {
    props,
    wrapper
  };
};

describe('LinearGradient check snapshot', () => {
  test('should match snapshot', () => {
    const { wrapper } = setup();
    expect(wrapper).toMatchSnapshot();
  });
});
