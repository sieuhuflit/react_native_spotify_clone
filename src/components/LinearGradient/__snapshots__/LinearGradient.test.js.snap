// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LinearGradient check snapshot should match snapshot 1`] = `ShallowWrapper {}`;
